qa/tasks/vstart_runner.py

```diff
@@ -168,11 +169,11 @@ def launch_subprocess(args, cwd=None, env=None, shell=True,
 
 try:
     from tasks.ceph_manager import CephManager
-    from tasks.cephfs.fuse_mount import FuseMount
-    from tasks.cephfs.kernel_mount import KernelMount
-    from tasks.cephfs.filesystem import Filesystem, MDSCluster, CephCluster
-    from tasks.cephfs.mount import CephFSMount
-    from tasks.mgr.mgr_test_case import MgrCluster
+    import tasks.cephfs.fuse_mount
+    import tasks.cephfs.kernel_mount
+    import tasks.cephfs.filesystem
+    import tasks.cephfs.mount
+    import tasks.mgr.mgr_test_case
```

Comment:
This seems like a rather opinionated change?

Reply:
Not at all, I had to do it because I need to update the internal variable to inject local implementation